docs/setup-tizen-environment.md

```diff
@@ -0,0 +1,7 @@
+# Setup Tizen Environment
```

Comment:
> But when you open Tizen Studio 1.3 it tries to update itself to 2.x version.
So you need to disable autoupdate and install TV Extension manually.
You can find instructions here. (See my comment on Andrew Witte's post)

Thanks for the advice.

Reply:
Seems to be a chaotic scenario @Blinnikov 😢

Reply:
Unfortunately it is. :disappointed: 
Samsung/Tizen forums are full of angry people who are indignant about the unclear development process.
By the way It looks I found a way to run apps from Tizen Studio 2.x on old devices `2.3+`.
I'll post on this in a while when I test all the scenarios.

Reply:
I've just posted an article on Medium describing the problem and the solution 🎉 :

[Developing for “old” Samsung TVs in Tizen Studio 2.x](https://medium.com/@ibazzva/developing-for-old-samsung-tvs-in-tizen-studio-2-x-5aa3f853db09)

Reply:
We don't really need Tizen SDK to develop the applications.
We need it to:
- Pack and Sign
- Run to Device
- Emulator

For the first two, we can do it without the Tizen SDK (ref: https://github.com/Samsung/vscode-extension-tizentv)
To Emulator we might be able to use only the contents from the "tv-extensions-X.X".

To Samsung Legacy SDK (Orsay?) we can use the stuff from cordova (https://github.com/Samsung/grunt-cordova-sectv/blob/master/tasks/packager/sectv-orsay.js)

Reply:
@dead You need the sdb part still for the extension (tizen studio/sdk) and they include it there
but: sdb is "basically" an adb, which is open source so you can go with it ;)
so it could be done (and it's done here)
- run your adb server with node, use socket and define the protocol
- implement the device discovery (or just map the devices via hand)
- implement the basic commands:
```
OPEN: (localId, destination = DESTINATIONS.SHELL) => {
    //Buffer("OPEN").readUInt32LE(0); // 0x4e45504f
    return SDB.pack(
      Buffer.from("OPEN").readUInt32LE(0),
      localId,
      0,
      destination
    );

   // and some command:
   SHELL: args => `shell:${args}:`,
   // 
   const cmd = SDB.COMMANDS.OPEN(0x42, SDB.DESTINATIONS.SHELL("0 vd_applist"));
  },

```
pack it, then send it thought your server, it will respond, make sure to have the talk going on with the client as it should and voila, in order to support older ones, just de-ref the 1.4 bridge and check the commands

:)  👍 
same goes for the emulator,
the packaging is easy as well, signing is in the same samsung like lazy level :) 
they mention security all over the place and they still keep using legacy tech on their flagship devices